webphenote-config/groups.yaml

```diff
@@ -0,0 +1,22 @@
+####
+#### Metadata on Monarch groups, derived from GOC groups
+#### See https://github.com/geneontology/go-site/tree/master/metadata
+####
+-
+  label: 'Monarch'
+  id: https://monarchinitiative.org
+-
+  label: 'LBL'
+  id: http://lbl.gov
```

Comment:
I copied my groups.yaml from the [go-site](https://github.com/geneontology/go-site/blob/master/metadata/groups.yaml) version. So maybe should be corrected in both?